src/viewmanager.cpp

```diff
@@ -67,6 +67,7 @@ ViewManager::ViewManager(DwarfModel *dm, DwarfModelProxy *proxy,
     m_add_tab_button->setToolTip(tr("Add an existing view. New views can be copied or created from the [Windows->Docks->Grid Views] dock."));
     draw_add_tab_button();
     setCornerWidget(m_add_tab_button, Qt::TopLeftCorner);
+    m_add_tab_button->setMinimumHeight(m_add_tab_button->minimumSizeHint().height());
```

Comment:
This will fix disapearing button
https://stackoverflow.com/questions/31116295/cornerwidget-disappears-when-there-isnt-any-tab

Reply:
Would changing the (vertical) size policy work? Something like `m_add_tab_button->setSizePolicy(QSizePolicy::Preferred, QSizePolicy::Minimum);`.

Reply:
No and don't know why. According to docs it's better solution, but it not working. I also tried some variations (Fixed, horizontal policy changes) - no success.

Reply:
This does not work with every style: adwaita still hide the button, but it works with fusion and windows styles.